site/docs/tools/iceberg_client.md

```diff
@@ -1,23 +1,18 @@
-# Spark
-
-Nessie can be used with Spark in different ways. These include:
-
-## Iceberg client
+# Iceberg Client
 
 Nessie works seamlessly with Iceberg in Spark2 and Spark3. Nessie is implemented as a custom iceberg
 [catalog](http://iceberg.apache.org/custom-catalog/) and therefore supports all features available to any Iceberg
-client. This includes Spark structured streaming, Presto and Flink. See the [iceberg docs](https://iceberg.apache.org)
+client. This includes Spark structured streaming, Presto, Flink and Hive. See the [iceberg docs](https://iceberg.apache.org)
 for more info.
 
 !!! note
-    You can follow along an interactive demo in a [Jupyter Notebook via Google Colab](https://colab.research.google.com/github/projectnessie/nessie-demos/blob/main/colab/nessie-iceberg-demo-nba.ipynb)
-    or in a [Jupyter Notebook via Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=colab%2Fnessie-iceberg-demo-nba.ipynb).
+    You can follow along an interactive demo in a [Jupyter Notebook via Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=colab%2Fnessie-iceberg-demo-nba.ipynb).
     
-    Detailed steps on how to set up Pyspark + Iceberg + Nessie with Python is available on
-    [Colab](https://colab.research.google.com/github/projectnessie/nessie-demos/blob/main/colab/nessie-iceberg-spark-setup.ipynb)
-    and [Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=colab%2Fnessie-iceberg-spark-setup.ipynb)
+    Detailed steps on how to set up Pyspark + Iceberg + Nessie with Python is available on [Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=colab%2Fnessie-iceberg-spark-setup.ipynb)
 
 
+## Spark
+
 To access Nessie from a spark cluster make sure the `spark.jars` spark option is set to include
 the [Spark 2](https://repo.maven.apache.org/maven2/org/apache/iceberg/iceberg-spark/{{ versions.iceberg }}/iceberg-spark-{{ versions.iceberg }}.jar)
```

Comment:
Do we still want to highlight Spark 2 here?
IIRC we threw away Spark2/Delta, not sure about Spark2/Iceberg though. @rymurr ?